css/index.css

```diff
@@ -53,12 +55,13 @@ table {
 
 html, body {
     height: 100%;
-    font-family: 'Roboto', sans-serif;
+    font-family: 'Roboto', sans-serif; 
+    font-size: 62.5%
 }
 
 h1, h2, h3, h4, h5 {
-    font-size: 18px;
-    margin-bottom: 15px;
+    font-size: 1.8rem;
+    margin-bottom: 1.5rem;
```

Comment:
Not that it's wrong, per say, but use either `%` or `px` for margins, paddings, width.

Use `rem` for fonts.